terraform-docs/action.yaml

```diff
@@ -50,16 +83,27 @@ runs:
             log-info "File ${README_FILE} does not exist. Terraform-docs will create new README.md file."
         fi
       continue-on-error: true
-    - id: run-terraform-docs
-      if: steps.validate-readme.outcome == 'success'
+
+    - id: generate-example-docs
+      if: steps.inject-config-files.outputs.examples-folder-exists == 'true'
+      uses: terraform-docs/gh-actions@aeae0038ed47a547e0c0fca5c059d3335f48fb25
+      with:
+        output-method: inject
```

Comment:
double `inject`, both from action and in config. you do not need `inject` here, it's the default ;-)

ref. <https://terraform-docs.io/user-guide/configuration/output/>

Reply:
Correct, but we need to set it as "empty string" if output.mode and output.file are defined in config file. Ref [action doc](https://github.com/terraform-docs/gh-actions?tab=readme-ov-file#inputs)